pkg/virt-handler/vm_test.go

```diff
@@ -2193,28 +2194,17 @@ var _ = Describe("VirtualMachineInstance", func() {
 			Expect(blockMigrate).To(BeTrue())
 			Expect(err).To(Equal(fmt.Errorf("cannot migrate VMI with non-shared HostDisk")))
 		})
-		table.DescribeTable("when host model labels", func(toDefineHostModelLabels bool) {
+		table.DescribeTable("when host model labels", func(withCapabilities bool) {
```

Comment:
you could pass the capability list directly, and avoid using a boolean argument.

Reply:
Changed